src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -312,10 +318,40 @@ InferenceEngine::Precision Convolution::fusedEltwisePrecision(const NodePtr& fus
 }
 
 const std::vector<impl_desc_type>& Convolution::getPrimitivesPriority() {
-    if (!cpuExperimental.count(EXPERIMENTAL_KEY_BRGCONV))
-        return Node::getPrimitivesPriority();
-
     std::vector<impl_desc_type> priorities = {
```

Comment:
May I ask you to avoid duplicating of the list?

Reply:
Duplicated list removed.